.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: python
 
 python:
- - "3.6"
+ - "3.7"
```

Comment:
This requires `dist: xenial`

Reply:
Thanks @simleo - I can try ```dist:xenial``` for travis, but it is unclear that it will fix the issue with the tests - I've been investigating previous versions of ```continuumio/miniconda3``` and pinning some of the packages versions without success for now